std_msgs/Header header
hand_object_detection_ros/HandDetection[] detections
